### ISSUE TITLE
Приложение не запускалось?

### ISSUE THREAD

**Issue by dvaschenko** [alkhachaturian/homework-2#1]
Подскажите, пожалуйста, Вы же не запускали приложение при выполнении этого домашнего задания? Или при запуске возникли проблемы, с которыми Вы не смогли справиться?

**Comment by dvaschenko** [comment]
Возможно, в процессе запуска было выброшено исключение неизвестного типа данных для BYTE[], решить указанную проблему можно 2-мя способами:
- изменить тип используемой базы данных на Postgresql
- внести изменения в корневой файл changelog.xml следующего вида:
    <property name="byte[].type" dbms="hsqldb" value="blob"/>
которое явно укажет какой тип использовать для хранения массива байтов при использовании HSQLDB - а именно тип BLOB в данном случае

**Comment by dvaschenko** [comment]
Задача проверена, решение принято

Несколько небольших дополнений/советов по решению:
- Для поля загрузки файла документа можно добавить 
````
showClearButton="true"
````
для добавления возможности очистки загруженного файла (нет в задании, но полезно для UX, как мне кажется)
- Для более полного соответствия заданию в application.properties можно передавать флаг 
````
jmix.ui.viewFileExtensions=
````
с пустыми параметрами. В данном случае загрузка файла будет начинаться принудительно без открытия предпросмотра в новой вкладке

**Comment by alkhachaturian** [comment]
> Подскажите, пожалуйста, Вы же не запускали приложение при выполнении этого домашнего задания? Или при запуске возникли проблемы, с которыми Вы не смогли справиться?

нет, все корректно отработало

**Comment by alkhachaturian** [comment]
> Возможно, в процессе запуска было выброшено исключение неизвестного типа данных для BYTE[], решить указанную проблему можно 2-мя способами:
> 
> * изменить тип используемой базы данных на Postgresql
> * внести изменения в корневой файл changelog.xml следующего вида:
> 
> ```
>     <property name="byte[].type" dbms="hsqldb" value="blob"/>
> ```
> 
> которое явно укажет какой тип использовать для хранения массива байтов при использовании HSQLDB - а именно тип BLOB в данном случае

да, я переделала настройки для постгреса, но в гит эти изменения не добавляла. В других заданиях тоже...

**Comment by dvaschenko** [comment]
В целом эти объяснения были дополнительными, решение принято, как написал выше.
